testdata/supportiety/server/meta.dyml

```diff
@@ -0,0 +1,18 @@
+#! {
+    // Bounded contexts are just folders in your domain folder.
+
+    name "Supportiety Server"
+    description "The supportiety server is responsible for managing support tickets."
+
+    license "Apache-2.0"
```

Comment:
SPDX?

Reply:
Ja genau, das hatte ich auch schonmal irgendwo festgehalten, werde das aber nochmal in `docs/README` ergänzen.